src/include/planner/plan_util.h

```diff
@@ -90,6 +90,12 @@ class PlanUtil {
         table_ids.insert(insert_node->GetTable()->GetOid());
         break;
       }
+      /*case PlanNodeType::POPULATE_INDEX: {
```

Comment:
I added this when I began development for completeness. But now, since there is a SeqScanPlan associated, I'm not sure if I should leave this case. I just tested and it completed fine. Should I leave it uncommented?